en/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: news_post
+title: "Support of Ruby 2.1 has ended"
+author: "usa"
+translator:
+date: 2017-04-01 00:00:00 +0000
+lang: en
+---
+
+We announce that all support of Ruby 2.1 series has ended.
+
+After the release of Ruby 2.1.10 at the end of March of the last year, the support of Ruby 2.1 series was in security maintenance phase.
+But, after one year has passed, this phase has ended.
+Therefore, at March 31st 2017, all support of Ruby 2.1 series has ended.
+Bug and security fixes from more recent Ruby versions will no longer be backported to 2.1, and no further patch release of 2.1 will be released.
+We highly recommend that you upgrade to Ruby 2.4 or 2.3 as soon as possible.
+
+## About currently supported Ruby versions
+
+* Ruby 2.4 series
```

Comment:
Newline missing below this line (else: "Ruby 2.4 series Currently in ..."

or definition list:

```
Ruby ...
: Currently...
  Foo...

```